src/core/execute.h

```diff
@@ -86,18 +87,11 @@ struct ExecStatus {
         int status;   /* as in sigingo_t::si_status */
 };
 
-typedef enum ExecCommandFlags {
-        EXEC_COMMAND_IGNORE_FAILURE   = 1 << 0,
-        EXEC_COMMAND_FULLY_PRIVILEGED = 1 << 1,
-        EXEC_COMMAND_NO_SETUID        = 1 << 2,
-        EXEC_COMMAND_AMBIENT_MAGIC    = 1 << 3,
-        EXEC_COMMAND_NO_ENV_EXPAND    = 1 << 4,
-} ExecCommandFlags;
-
 /* Stores information about commands we execute. Covers both configuration settings as well as runtime data. */
 struct ExecCommand {
         char *path;
         char **argv;
+        bool is_ex_property; /* whether this command came from an ExecXYZEx= property */
```

Comment:
we shouldn't distinguish this. I am pretty sure we should just expose all props in both ExecXZY= and ExecXYZEx= foo, even if that's redundant. The latter would be the new, and comprhensive prop and the old one would be the legacy prop that hides some info, but that'd be OK. It comes of course at the price that we send a bit too much data, but I think that's fine